packages/gradient_boosting_model/gradient_boosting_model/pipeline.py

```diff
@@ -0,0 +1,61 @@
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import OrdinalEncoder
+from sklearn.ensemble import GradientBoostingRegressor
+from sklearn.pipeline import Pipeline
+from feature_engine.categorical_encoders import RareLabelCategoricalEncoder
```

Comment:
Make sure we are using latest version of Feature-engine. Shouldn't affect the rarelabelencoder tho.

Reply:
We're using v 3.0 which I believe is the latest